README.md

```diff
@@ -1,2 +1,149 @@
-# hsdo
-HAProxy Service Discovery Operator
+# haproxy-service-discovery-orchestrator
+Orchestrate Service Discovery for HAProxy
+
```

Comment:
```suggestion

We are currently using HSDO to load-balance our VOD traffic, between CDNs and our origins.
We have tested this platform with tens of HAProxy instances, reaching up to 200Gbps traffic.
We are using Spot instances to run HAProxy with HSDO, using multiple AZs, but by optimizing traffic through AZ (because inter-AZ traffic is extremely expensive).

```